tbf-challenge/teamwork-be
# record number of likes

#### Pre-Submission Checklist






#### Checklist:





#### What does this PR do?


https://github.com/tbf-challenge/teamwork-be/issues/307
#### How should this be manually tested?


1. Start the server with `npm run dev`
2. You will see with the logs that the migration has been added to the database

##### Any background context you want to provide?



#### What are the relevant issues?

https://github.com/tbf-challenge/teamwork-be/issues/307
